plugins/default/lib/mongodb_health.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python
+import argparse
+import sys
+import traceback
+from collections import Counter
+
+import openshift
+import nagios
+from mongodb import ReplsetStatuses
```

Comment:
Do we really need this to be defined in another file?

Unless we're planning to have more MongoDB checks that need to know about ReplsetStatuses and could not import it from `mongodb_health`, there's no need for `mongodb.py` and `enum.py`, as there are very specific to this check.

Think if this was Bash... we probably wouldn't start breaking files into a lot of small files and sourcing them from other scripts.

I'd argue that when we look at the list of files under `plugins/default/lib`, we want to see a list of checks, and any noise there is just distracting. I'd strive to keep the noise at a minimum.
Apart from checks, we have there:

- Tests (those our brains can filter out for they all match `test_*.py`)
- `nagios.py` (should be used/imported by all checks)
- `openshift.py` (should be used by most checks)